all-repos.yaml

```diff
@@ -242,3 +242,5 @@
 - https://github.com/igrr/astyle_py
 - https://gitlab.com/codementors/pre-commit/add-issue-number-to-conventional-commit
 - https://github.com/numpy/numpydoc
+- https://github.com/c0m1c5an5/gitlab-ci-tools
```

Comment:
this could probably depend on `shellcheck-py` and then you wouldn't require manual setup by the user to have shellcheck available which makes at least one of the hooks here not up to standards (as it's essentially a `language: system` hook)

Reply:
Yes, thank you for the suggestion. `shellcheck-py` fits perfectly. Yet other hooks (`gitlab-ci-fmt` and `gitlab-ci-lint`) also rely on local system utilities, so I am in a process of rewriting them to fit this requirement.